v03_pipeline/lib/annotations/mito.py

```diff
@@ -90,12 +66,26 @@ def rsid(ht: hl.Table, **_: Any) -> hl.Expression:
     return ht.rsid.find(lambda x: hl.is_defined(x))
 
 
-def gt_stats(ht: hl.Table, sample_lookup_ht: hl.Table, **_: Any) -> hl.Expression:
+def gt_stats(
```

Comment:
this is just a refactor to make the mito gt_stats work/look the same as the snv_indel one (which was re-worked for performance issues that we don't have for mito).